contracts/Super1155.sol

```diff
@@ -766,7 +769,7 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
 
     // Retrieve the item's group ID.
     uint256 shiftedGroupId = (_id & GROUP_MASK);
```

Comment:
Can this be safely deleted if you're using `_id` directly instead?

Reply:
No. This is a mistake actually, it needs to be kept as it was.